css/style.css

```diff
@@ -0,0 +1,68 @@
+   body{
+        background-color: lightblue;
+        text-align: center;
+        
+      }
+
+      h1 {
+        color: blue;
+        align-items: center;
+      }
```

Comment:
- [ ] Font family and font size properties should be added so that the text size and length can be controlled.

Reply:
- Let's be more friendly by making the comments more like suggestions, this way the person we are reviewing the code feels confident and likely to improve their work with our requested changes.